### PR TITLE
Create v_ksm_alternate_regions.sql

### PR DESCRIPTION
**Pull request by ssocite**
Change of KARP search to "LIKE" instead of "="

### CODE REVIEW

File: views/v_ksm_alternate_regions.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+       comm.stop_dt,
+       comm.committee_code,
+       tms.short_desc,
+       comm.start_dt,
+       comm.committee_status_code,
+       comm.committee_title,
+       comm.geo_code,
+       comm.xcomment
+FROM  committee comm
+Inner Join rpt_pbh634.v_entity_ksm_degrees deg on deg.ID_NUMBER = comm.id_number
```
Comment (phively) [inline views/v_ksm_alternate_regions.sql]: This view isn't used in the select; remove for speed

File: views/v_ksm_alternate_regions.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Left Join TMS_COMMITTEE_TABLE tms on tms.committee_code = comm.committee_code
+where  comm.committee_status_code = 'C'  
+and comm.xcomment = 'KARP'
```
Comment (phively) [inline views/v_ksm_alternate_regions.sql]: Please use:
Like '%KARP%'
to catch comments with additional information entered.

File: views/v_ksm_alternate_regions.sql [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Order by tms.short_desc ASC)
+
+Select deg.ID_NUMBER,
```
Comment (phively) [inline views/v_ksm_alternate_regions.sql]: Should be alternate_region.id_number
Actually let's take out the degree view entirely; it's possible non-alumni will be coded in alternate address regions

File: views/v_ksm_alternate_regions.sql [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+       house.HOUSEHOLD_ZIP,
+       house.HOUSEHOLD_GEO_CODES,
+       house.HOUSEHOLD_GEO_PRIMARY,
+       house.HOUSEHOLD_GEO_PRIMARY_DESC,
+       house.HOUSEHOLD_COUNTRY,
+       alternate_region.short_desc,
+       alternate_region.start_dt,
+       alternate_region.committee_status_code,
+       alternate_region.xcomment
+From rpt_pbh634.v_entity_ksm_degrees deg
```
Comment (phively) [inline views/v_ksm_alternate_regions.sql]: Replace with Entity table
You can use entity.report_name and entity.institutional_suffix; no need for any of the other fields

File: views/v_ksm_alternate_regions.sql [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+Left Join rpt_pbh634.v_ksm_prospect_pool prospect on prospect.ID_NUMBER = deg.ID_NUMBER
```
Comment (phively) [inline views/v_ksm_alternate_regions.sql]: Remove -- never use this view if you don't have to! Very slow and not needed for institutional_suffix (already in Entity)

File: views/v_ksm_alternate_regions.sql [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+Left Join rpt_pbh634.v_entity_ksm_households house on house.ID_NUMBER = deg.ID_NUMBER
+Inner Join alternate_region on alternate_region.id_number = deg.id_number
```
Comment (phively) [inline views/v_ksm_alternate_regions.sql]: Inner Join alternate_region on alternate_region.id_number = entity.id_number

File: views/v_ksm_alternate_regions.sql [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+Order By deg.REPORT_NAME ASC
```
Comment (phively) [inline views/v_ksm_alternate_regions.sql]: Order By entity.report_name Asc

File: views/v_ksm_alternate_regions.sql (hunk 1 above)
Comment (ssocite) [reply views/v_ksm_alternate_regions.sql]: Removed v_entity_ksm_degrees

File: views/v_ksm_alternate_regions.sql (hunk 2 above)
Comment (ssocite) [reply views/v_ksm_alternate_regions.sql]: Now using '%KARP%' as contains.

File: views/v_ksm_alternate_regions.sql (hunk 3 above)
Comment (ssocite) [reply views/v_ksm_alternate_regions.sql]: I took out the degree view and replaced it with entity.

File: views/v_ksm_alternate_regions.sql (hunk 4 above)
Comment (ssocite) [reply views/v_ksm_alternate_regions.sql]: Used entity report name and institutional suffix from the entity table.

File: views/v_ksm_alternate_regions.sql (hunk 6 above)
Comment (ssocite) [reply views/v_ksm_alternate_regions.sql]: Inner Joined Alternate_region and entity_ksm_households. Prospect pool has been removed!

File: views/v_ksm_alternate_regions.sql (hunk 7 above)
Comment (ssocite) [reply views/v_ksm_alternate_regions.sql]: Ordered by entity report.

**Comment by ssocite** [comment]
Hello Paul, 

I made the modifications and re-uploaded through github desktop. The new version takes significantly less time to extract the data. Thanks for the recommendations

!

**Pull request merged**